content/docs/operating/configuration.md

```diff
@@ -703,6 +707,54 @@ paths:
 
 Serverset data must be in the JSON format, the Thrift format is not currently supported.
 
+### `<triton_sd_config>`
+CAUTION: Triton SD is in beta: breaking changes to configuration are still
+likely in future releases.
+
+[Triton](https://github.com/joyent/triton) SD configurations allow retrieving
+scrape targets from [Container Monitor](https://github.com/joyent/rfd/blob/master/rfd/0027/README.md)
+discovery endpoints.
+
+Prometheus will periodically check the discovery endpoint(s) for running
+containers and create a target for them.
+
+The following meta labels are available on targets during relabeling:
+
+* `__meta_triton_machine_id`: the UUID of the target container
+* `__meta_triton_machine_alias`: the alias of the target container
+* `__meta_triton_machine_image`: the target containers image type
+* `__meta_triton_machine_server_id`: the server UUID for the target container
+
+
+See below for the configuration options for Triton discovery:
```

Comment:
This appears to be the header for every other SD configuration options section, so it might appear somewhat inconsistent if it is removed. Am I missing something that makes the Triton section different?